src/sickle.ts

```diff
@@ -380,7 +384,14 @@ class Annotator {
     try {
       return getJSDocAnnotation(comment);
     } catch (e) {
-      this.fail(e.toString());
+      this.diagnostics.push({
+        file: this.file,
+        start: node.getFullStart() + pos,
+        length: end - pos,
+        messageText: e.message,
+        category: ts.DiagnosticCategory.Error,
+        code: 0,
```

Comment:
code:0 is required by the type.

The reason to reuse ts.Diagnostic is that it lets the user of sickle merge sickle errors with TS errors and print them all with the same interface.